fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -86,266 +81,6 @@ public class OnlineIndexerSimpleTest extends OnlineIndexerTest {
     private static final Pattern SCAN_RECORDS_PATTERN = TestHelpers.eventCountPattern(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_SCANNED);
     private static final Pattern BUILD_RANGES_PATTERN = TestHelpers.eventCountPattern(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RANGES_BY_COUNT);
 
-    @Test
```

Comment:
It seems like these tests should also stick around until the code is deleted.

Reply:
✅